src/ariel-os-esp/src/gpio.rs

```diff
@@ -111,16 +112,14 @@ impl From<DriveStrength> for esp_hal::gpio::DriveStrength {
 
 impl ariel_os_embassy_common::gpio::FromDriveStrength for DriveStrength {
     fn from(drive_strength: ariel_os_embassy_common::gpio::DriveStrength<Self>) -> Self {
-        use ariel_os_embassy_common::gpio::DriveStrength::*;
-
         // ESPs are able to output up to 40 mA, so we somewhat normalize this.
         match drive_strength {
-            Hal(drive_strength) => drive_strength,
-            Lowest => Self::_5mA,
-            Standard => Self::_10mA,
-            Medium => Self::_10mA,
-            High => Self::_20mA,
-            Highest => Self::_40mA,
+            ariel_os_embassy_common::gpio::DriveStrength::Hal(drive_strength) => drive_strength,
```

Comment:
IMO this decreases readability. Could we instead move the imports to file level? At lease `ariel_os_embassy_common::gpio::DriveStrength`?